Mail2Bug/Email/EWS/EWSIncomingMessage.cs

```diff
@@ -33,7 +36,9 @@ public EWSIncomingMessage(EmailMessage message)
                     MeetingRequestSchema.Start,
                     MeetingRequestSchema.End
                 ));
-            
+
+            message.TryGetProperty(conversationId, out _conversationId);
```

Comment:
if this fails, _conversationId is going to be null. ConversationGuid is going to throw an exception then. not sure what ConversationIndex returns if property doesn't exist though.